awx/playbooks/library/indirect_instance_count.py

```
@@ -52,6 +55,91 @@ def list_collections(artifacts_manager=None):
+        if match:
+            version_str = match.group(1)
+            try:
+                versions.append(Version(version_str))
+            except InvalidVersion:
+                # Skip files with invalid version strings
+                pass
+
+    return versions
+
+
+def find_external_query_with_fallback(namespace, name, installed_version, display=None):
```

Ruff (0.15.2)

[warning] Unused function argument: `display`

(ARG001)